lib/vsc/filesystem/operator.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: latin-1 -*-
+#
+# Copyright 2009-2022 Ghent University
```

Comment:
copy & paste, it's a dangerous thing :wink:

Reply:
Actually, I've been forced against my will to set this copyright by the unit tests.